### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by DmytroBondariev**

### CODE REVIEW

**Comment by DmytroBondariev** [comment]
1)Link to current authenticated user page and hover with links to other pages
image

2)Position/TaskType/Page visits counters in first block of home page. 
image


3)Task creation link in second block of home page
image

4)Workers page with search form, detail links and tasks assigned count, create button(Pagination included)

image

5)Worker detail page with delete/update buttons and tasks assigned(link per task)

image

6) Worker update form:

image


7) Worker create form :

image

8)Task list page with create button(available only for users with CEO/Project Managet position) and search field(Pagination included)

image

9) Task detail page with update button, assign/misassign to current user button and list with assigned workers(link per worker)

image

10) Task update page

image

11) Task create page

image

**Comment by DmytroBondariev** [comment]
task_manager_models_scheme

File: task_manager/forms.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm, \
+    AuthenticationForm, UsernameField
```
Comment (Grayder0152) [inline task_manager/forms.py]: Use `()` instead of `\`

File: task_manager/models.py [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+    class Meta:
+        verbose_name = "worker"
+        verbose_name_plural = "workers"
+        ordering = ["-position"]
+
+    def __str__(self):
+        return f"{self.username} " \
+               f"({self.first_name} {self.last_name}, " \
+               f"position: {self.position.name if self.position else None})"
```
Comment (Grayder0152) [inline task_manager/models.py]: use `()` instead of `\`

File: task_manager/urls.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
+from django.urls import path
+
+from task_manager.views import IndexView, WorkerListView, UserLoginView, \
+    TaskListView, WorkerDetailView, TaskCreateView, TaskUpdateView, \
+    TaskDetailView, WorkerCreateView, WorkerUpdateView, \
+    ToggleAssignToTaskView, WorkerDeleteView, UserLogoutView
```
Comment (Grayder0152) [inline task_manager/urls.py]: Use `()` instead of `\`

File: task_manager/views.py [hunk 4]
```diff
@@ -0,0 +1,162 @@
+from django.contrib.auth import logout
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.contrib.auth.views import LoginView, LogoutView
+from django.http import HttpResponseRedirect
+from django.shortcuts import render, redirect
+from django.urls import reverse_lazy
+from django.views import generic
+
+from task_manager.forms import UserLoginForm, \
+    WorkerSearchForm, TaskSearchForm, TaskForm, WorkerForm
```
Comment (Grayder0152) [inline task_manager/views.py]: Use `()` instead of `\`

File: task_manager/views.py [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+    template_name = "pages/worker_confirm_delete.html"
+
+
+class WorkerListView(generic.ListView):
+    model = Worker
+    paginate_by = 6
+    template_name = "pages/worker_list.html"
+
+    def get_context_data(self, *, object_list=None, **kwargs):
+        context = super(WorkerListView, self).get_context_data(**kwargs)
```
Comment (Grayder0152) [inline task_manager/views.py]:
```suggestion
        context = super().get_context_data(**kwargs)

```

**Review by Grayder0152 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**